tests/Agent/UnitTests/NewRelic.Agent.Extensions.Tests/Lambda/LambdaEventHelpersTests.cs

```diff
@@ -29,6 +29,39 @@ public class LambdaEventHelpersTests
     private const string W3CTraceParentPayload = "00-da8bc8cc6d062849b0efcf3c169afb5a-7d3efb1b173fecfa-01";
     private const string W3CTraceStatePayload = "33@nr=0-0-33-2827902-7d3efb1b173fecfa-e8b91a159289ff74-1-1.23456-1518469636035";
 
+    // I don't like hardcoding the same values for the tracing data in this JSON string as are already defined above, but
+    // having to escape everything else to make string interpolation possible is gross.
+    private const string SnsBodyJson = @"{
```

Comment:
I learned a new thing today while working on the integration tests -- you can convert this to a "raw string" and get rid of all the double quotes. 

There's a context menu fix available for me, but it might be coming from ReSharper. 

If you change this line to 
```
private const string SnsBodyJson = """
```` 
and change the closing (down around line 63) to `""";` you can drop all the double quotes. 

And if you want to get really fancy, you can, indeed, use string interpolation pretty cleanly here -- use 
```
private const string SnsBodyJson = $$"""
```
and then use double-curly for the string interpolation, eg. 
```
"{{W3CTraceParentKey}}": {
```

Refer to [this](https://github.com/newrelic/newrelic-dotnet-agent/blob/feature/lambda-support/tests/Agent/IntegrationTests/IntegrationTests/RemoteServiceFixtures/AwsLambda/LambdaApplicationLoadBalancerRequestTriggerFixture.cs#L62) from my recent PR for more details.

Reply:
Wow, that's really great.  Thanks!  And done.